include/AnalyzerVisitors/GeometricInfo.hh

```diff
@@ -0,0 +1,185 @@
+#ifndef _GEOMETRICINFO_HH
+#define	_GEOMETRICINFO_HH
+
+#include <global_constants.hh>
+#include "VizardTools.hh"
+#include <Tracker.hh>
+#include <Analyzer.hh>
```

Comment:
If the purpose is making Analyzer disappear, shouldn't we move the constants/variables/functions we need in Analyzer.hh into something different? I created AnalyzerTools for this, but any other choice is equally fine for me